graphql-ballerina/http_service.bal

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+
+service class HttpService {
+    private Engine engine;
+
+    public isolated function init(Engine engine) {
+        self.engine = engine;
+    }
+
+    isolated resource function get .(http:Caller caller, http:Request request) {
+        http:Response response = handleGetRequests(self.engine, request);
+        var result = caller->respond(response);
```

Comment:
we can simply do return response right as below 

isolated resource function post .(http:Caller caller, http:Request request) returns http:Response  {
        return handlePostRequests(<@untainted>self.engine, request);
}

Reply:
Tried to do that, but then the tests started to hang. 
Will add a `TODO` comment in the code so that we can change this once that is fixed?

Reply:
Yeah, no worries :) 

@chamil321 FYA